docs/src/modules/spring/pages/service-to-service.adoc

```diff
@@ -1 +1,47 @@
 = Service to Service Eventing
+
+include::ROOT:partial$include.adoc[]
+
+
+Kalix provides brokerless at-least-once event delivery across Kalix services through the Service to Service eventing.
+
+The source of the events is an xref:event-sourced-entities.adoc[Event Sourced Entity], that the service provides
+a Service to Service publisher for.
+
+Note:: For eventing from an entity inside the same Kalix service as the consuming component, use regular
+xref:actions-publishing-subscribing.adoc[Subscription to the entity] instead of Service to Service eventing.
+
+== Event Producer
+
+The event producer controls which entity to publish events for. Each entity published is identified by a stream `id` so
+that one Kalix service can publish more than one of the entity types it contains.
+
+[source,java,indent=0]
+.src/main/java/customer/api/CustomerEventsService.java
+----
+include::example$spring-eventsourced-customer-registry/src/main/java/customer/api/CustomerEventsService.java[tag=producer]
+----
+<1> `@Subscribe.EventSourcedEntity` identifying which event sourced entity to publish events for
+<2> Ignore any event types not handled by a method and move on with the event stream, rather than fail which is the default
+<3> `@Publish.Stream` marks the public identifier for consumers of this stream
+<4> An ACL annotation, allowing access from other Kalix services, but not the public internet
+<5> All methods on the service are transformer methods for turning internal event message types into public API message types for other services to consume
```

Comment:
```suggestion
----
<1> Identifying which event sourced entity to publish events for.
<2> Ignore any event types not handled by a method and move on with the event stream, rather than fail which is the default.
<3> Public identifier for consumers of this stream.
<4> Allowing access from other Kalix services, but not the public internet.
```

Reply:
Thanks for the suggestion. I think it's better so.
However, point 5 is being dropped in this suggestion. So I will create another one to apply.